frontend/app/templates/about-us.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="ui grid">
+    <div class="eight wide column" style="text-align:justified">
+        <h1 style="text-align:Justified">About Us</h1>
+        <p>Badgeyay is a Badge generator with a simple web UI to add data and generate printable badges in a PDF.
+
+            The user can:
+            <ul>
+                <li>Choose the size of the badges</li>
+                <li>Choose the background of the badges and upload the logo and the background image</li>
+                <li>Upload a CSV file or manually enter CSV data as: name, type of attendee, designation, nick/handle,
+                    organization/project</li>
+            </ul>
+        </p>
+        <h1 style="text-align:Justified">About Fossasia</h1>
+        <div>
+            <p>FOSSASIA is an organization developing software applications for social change using a wide-range of
+                technologies. It was established 2009. Projects range from Free and Open Source software, to design,
+                graphics and hardware. We also organize and participate in conferences, meetups and code camps. The
+                annual FOSSASIA OpenTechSummit is one of the largest gatherings of developers in Asia and took place in
+                Vietnam, Cambodia and Singapore. Please join us and start contributing to our projects on <a src="https://github.com/fossasia">Github</a>.</p>
+        </div>
+    </div>
+    <div class="eight wide column">
+        <img src="https://github.com/fossasia/badgeyay/raw/development/api/docs/installation/demo.gif">
+    </div>
+    <div class="sixteen wide column">
+        You may reach out to us at <a href="mailto:badgeyayofficial@gmail.com">badgeyayofficial@gmail.com</a>
```

Comment:
Try to use binding here, because in the near future if we need to change the address then we have to manually change it in places.